src/Enterspeed.Source.UmbracoCms/App_Plugins/Enterspeed.Dashboard/Dashboard.Sub.Controllers/enterspeed-failedjobs.controller.js

```diff
@@ -16,7 +16,7 @@
 
     function getConfiguration() {
         vm.loadingConfiguration = true;
-        dashboardResources.getEnterspeedConfiguration()
```

Comment:
Temp workaround for existing issue being fixed elsewhere

Reply:
Check, I guess we should remove this and as you mention fix this in another PR